### ISSUE TITLE
Nearby Venue Notifications

### ISSUE THREAD

**Issue by volkzayaz** [jmatheson/OnMe-Backend#68]
[josh]

I am not getting the location based notification that I am near a venue where a drink is waiting. I tested with Occidental yesterday. This did work in the old version of the app. My location services are enables for always on.

**Comment by volkzayaz** [comment]
Please introduce new API for receiving non paginated list of **received** Beverages which are in status `New` or `Redeem`. These are needed for proper set up of Geofence Notification

**Comment by kostya-p-dev** [comment]
Done. I added method GET api/new-redeem-drinks/  section Drink in doc

**Comment by volkzayaz** [comment]
http://app.onmetheapp.com/api/new-redeem-drinks/
Bearer 06cbd2bea92abde7745db5b464b32f40

returned 110 drinks. Most of them are not related to this user at all. Please make sure that only **received** beverages are returned.

Additionally response lacks corresponding `Venue` which is crucial for this method, because it's used for settings up geofences

**Comment by kostya-p-dev** [comment]
Done . I added venue info to response

**Comment by volkzayaz** [comment]
@jidulberger, @jmatheson. This is fixed in 0.0.18. I bough each of you a drink in your favorite Venues. Hope they are not too far away. I restored Permission for Location logic from v1. App requests "Always" location upon opening drink details that has not been redeemed. Make sure you either visit this screen or turn on "Always" location manually.

Please verify and close ticket.

**Comment by jidulberger** [comment]
We also used to warn the user when she was attempting to redeem a drink at a location she was not close to. I'll open a second issue for that.

**Comment by jmatheson** [comment]
I won't be able to test it. The venue is far away :)

**Comment by jmatheson** [comment]
I have manually switch my usage to be always on. I was never prompted to change to always on, so that is probably a different issue.

**Comment by volkzayaz** [comment]
@jmatheson, bought you a drink at Black Toast as well.

**Comment by volkzayaz** [comment]
@kostya-p-dev, currently server returns Venue's coordinates as strings. Please, make them return as numbers.

**Comment by kostya-p-dev** [comment]
Done . http://app.onmetheapp.com/api/new-redeem-drinks/  - will return numbers.

**Comment by jidulberger** [comment]
I just tested, still does not seem to work.

**Comment by jmatheson** [comment]
This works for me. Has worked several times in the last couple of days.
This feedback from Vlad came through today
"I just tested it by physically walking by a bar. I did receive notification as well.
The code has 100 meters radius, though notification popped up only when I was 20 meters away from venue"

100 meters seems right to me.

Let me know what else we can do to help you test @jidulberger 
Are there other venues we can buy you drinks at?
